JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt

```diff
@@ -52,7 +52,7 @@ fun PostCardTop(post: Post, modifier: Modifier = Modifier) {
         val imageModifier = Modifier
             .heightIn(min = 180.dp)
             .fillMaxWidth()
-            .clip(shape = MaterialTheme.shapes.medium)
+            .clip(shape = MaterialTheme.shapes.large)
```

Comment:
I think this should also apply to the `PostHeaderImage` in `PostContent.kt` ?

Reply:
sure...fixed in updated commit